### DISCUSSION TITLE
No effect with --no-cache

### DISCUSSION THREAD

**Discussion by Rechdan** [parcel-bundler/parcel#4916] [General] [answered]
I can't disable cache, here is my script: `parcel watch ./src/server/index.ts --target server --no-autoinstall --no-cache`

**Comment by DeMoorJasper** [comment, marked as answer]
`--no-cache` just means we don't reuse items from the cache on a rebuild, it does not mean we totally do not create a cache nor use it.

We use the cache to transmit files between workers if this flag is used. But it does not really behave as a cache anymore.

**Reply by DeMoorJasper** [reply]
This is true when using Parcel 2, with Parcel 1 it totally disables it. This is mainly due to memory issues with parcel 1

**Reply by Rechdan** [reply]
I see, thats good to know, thanks!
